docs/content.zh/docs/dev/table/types.md

```diff
@@ -191,6 +155,36 @@ For the Python Table API, those types are available in `pyflink.table.types.Data
 {{< /tab >}}
 {{< /tabs >}}
 
+The default planner supports the following set of SQL types:
+
+| Data Type        | Remarks for Data Type                              |
+|:-----------------|:---------------------------------------------------|
+| `CHAR`           |                                                    |
+| `VARCHAR`        |                                                    |
+| `STRING`         |                                                    |
+| `BOOLEAN`        |                                                    |
+| `BINARY`         |                                                    |
+| `VARBINARY`      |                                                    |
+| `BYTES`          |                                                    |
+| `DECIMAL`        | Supports fixed precision and scale.                |
+| `TINYINT`        |                                                    |
+| `SMALLINT`       |                                                    |
+| `INTEGER`        |                                                    |
+| `BIGINT`         |                                                    |
+| `FLOAT`          |                                                    |
```

Comment:
Adjust some lines in the docs/content.zh/docs/dev/table/types.md to ensure that this page is consistent with the English version. Please see [here](https://github.com/apache/flink/pull/18813/commits/da1c918a08dfcb50e27ffa0b54b36c4e711b460d)